### PR TITLE
DX-1528: Fix Headers

### PR DESCRIPTION
**Pull request by arebra**
Re-worked and added a lot of headers to make the menu and the pages easier to navigate.

### CODE REVIEW

**Review by trieutran91 — APPROVED** [review]

File: _includes/reversal.md [hunk 1]
```diff
@@ -55,6 +57,8 @@ Content-Type: application/json
 The `reversal` resource contains information about the newly created reversal
 transaction.
 
+## Reversal Request
```
Comment (trieutran91) [inline _includes/reversal.md:60]:
```suggestion
## Reversal Response
```

File: payment-instruments/vipps/after-payment.md [hunk 2]
```diff
@@ -240,16 +238,20 @@ Authorization: Bearer <AccessToken>
 Content-Type: application/json
 ```
 
+## GET Response
```
Comment (trieutran91) [inline payment-instruments/vipps/after-payment.md:241]:
```suggestion
## Cancel Response
```

File: payment-instruments/vipps/after-payment.md [hunk 3]
```diff
@@ -284,13 +290,17 @@ Authorization: Bearer <AccessToken>
...
```
Comment (trieutran91) [inline payment-instruments/vipps/after-payment.md:293]:
```suggestion
## Reversal Response
```

**Comment by arebra** [comment]
Fixed in other branches, and we have a fresher branch with the 137 error. Closing to clean up.

**Pull request closed without merging**